apps/academy/src/pages/tracks/nft-solidity/5.mdx

```diff
@@ -172,55 +172,19 @@ creating a way to handle wallet connections.
 
 ### The Main Application
 
-Install RainbowKit and its dependencies:
-
-```bash
-npm install @rainbow-me/rainbowkit wagmi viem
-```
-
-To get this setup we need to start working on the `page.js` file in the
-`/frontend/src/app` directory. The file should look something like this from our
-initial setup.
-
-```jsx
-import "@/styles/globals.css";
-
-export default function App({ Component, pageProps }) {
-  return (
-    <Component {...pageProps} />
-  );
-}
-
-export default MyApp;
-```
-
-We need to add some additional imports to create our configurations.
-
-{/* n.b. see below in following code block // `import { useEffect, useState } from "react";` MARKKOS' NOTE: I would remove this `import` and consider using/moving it to `index.js` imports explanation - go to line :238 and see comments re: sidedrawer needed */}
-
-```jsx
-import "@/styles/globals.css";
-// new block of imports:
-import "@rainbow-me/rainbowkit/styles.css";
-import { getDefaultWallets, RainbowKitProvider } from "@rainbow-me/rainbowkit";
-import { WagmiConfig, createConfig, configureChains } from "wagmi";
-import { polygonMumbai } from "wagmi/chains";
-
-import { alchemyProvider } from "wagmi/providers/alchemy";
-import { publicProvider } from "wagmi/providers/public";
-import { useEffect, useState } from "react";
-// end of new block of imports
+RainbowKit and its dependencies are already installed and ready-to-go.
 
-// the rest of the function MyApp code...
-```
+To get this setup we need to start working on the `pages/_app.tsx` file in the
+`/frontend/pages` directory. The file should look something like this from our
+initial setup:
```

Comment:
Good catch!

Thanks for including the code you see at this point, very helpful.

I added this code to the lesson, and updated the instructions to match. And since hooks are no longer used here, I moved the Hooks intro further down in the lesson.

https://github.com/Developer-DAO/academy-turbo/pull/131/commits/1c1ecae3103f1c6c82195e90b8c1e9a357d2b61d